.gitignore

```diff
@@ -40,4 +40,3 @@ src/config.h
 # Ignore the build directory
 build/*
 !build/.gitkeep
-
```

Comment:
not sure what the deal is with that '-'. Accidentally removed it. Is it necessary?